pwndbg/color/theme.py

```diff
@@ -32,5 +41,11 @@ def add_param(
     )
 
 
-def add_color_param(name: str, default: Any, set_show_doc: str) -> Parameter:
-    return config.add_param_obj(ColorParameter(name, default, set_show_doc, scope=Scope.theme))
+def add_color_param(name: str, default: Any, set_show_doc: str) -> ColorParameter:
+    color_parameter = ColorParameter(name, default, set_show_doc, scope=Scope.theme)
+
+    config.triggers[name].append(lambda: color_parameter.update_color_function())
```

Comment:
```suggestion
    config.triggers[name].append(color_parameter.update_color_function)
```
Shouldn't this be like this?